### ISSUE TITLE
Trees not spawning (1.18.2)

### ISSUE THREAD

**Issue by ItsGlace01** [ferreusveritas/Dynamic-Trees#717]
Trees dont spawn when loading a new world. This started when i updated all my mods on curseforge. So the trees before i updated are still there. Its just new ones are not spawning when loading chunks or loading a new world.
Heres smth i found on logs that i think might be causing the problem.

Error whilst loading type "SoilProperties" with name "dynamictrees:rooted_moss": [primitive_soil] Could not find block for registry name 'minecraft:rooted_moss'.

After that theres just stuff like 

 Exception loading blockstate definition: dynamictrees:blockstates/rooty_rooted_moss.json: java.io.FileNotFoundException: dynamictrees:blockstates/rooty_rooted_moss.json

And goes on with every block that has the name "root" in it.

**Comment by Harleyoc1** [comment]
Both of those issues should be fixed as of the latest update but would not have been related. Do you have any other mods installed that mess with worldgen?

**Comment by steverie119** [comment]
i have been having the same issue, namely spruce and dark oak trees

**Comment by Meshinator69** [comment]
I am also having this issue, but it's sporadic. It's difficult to replicate - sometimes the forests generate perfectly normal, and then once this "begins," very few trees generate again. Forests or Savannas for example are all completely empty. I was able to replicate this with just Biomes O plenty, Dynamic Trees and Terrablender.

Edited to Add, I do have Dynamic Trees for BOP as well.

**Comment by xFirefalconx** [comment]
Same issue here, I am going to investigate this further on my end as far as I can. I have a minecraft world where old trees exist, but all newly generated chunks are void of trees. A new map that i created for debugging... well doesn't generate them.
Affected are: Birch trees, Dark Oak trees, possibly even more. 
I also updated my game so I will edit this once I tested it again on whether the issue still persists. 
my latest.log: 
[latest.log](https://github.com/DynamicTreesTeam/DynamicTrees/files/11301503/latest.log)


EDIT: The normal overworld is fine now, there's an issue with Chunk by Chunk spawned chunks though, maybe it's a problem on their side though.
